qemu/tests/numa_prealloc_threads.py

```diff
@@ -7,6 +7,18 @@
 from avocado.utils import process
 
 
+def check_affinity(affinity, cmd_taskset, test):
+    """
+    :param affinity: the cpu affinity
+    :param cmd_taskset: the taskset command
+    :param test: QEMU test object
+    """
+    output = process.getoutput(cmd_taskset)
+    if not re.search(affinity, output):
+        test.fail("The affinities %s and %s do not match!"
+                  % (affinity, output))
```

Comment:
Suggest:
```
def check_affinity(affinity, cmd_taskset, stage, test):
    output = process.getoutput(cmd_taskset)
    actual_affinity = re.search("%s affinity list: (%s)" % (stage, affinity), output).group(1)
    if actual_affinity != affinity:
        test.fail("Expect %s cpu affinity '%s', but get '%s'" % (stage, affinity, actual_affinity))
```

Reply:
@yanan-fu why is the `stage` param needed here? This function is always called to check the **current** cpu-affinity list

Reply:
Refer to the comment below,  check if qemu main thread use the correct affinity can reuse this function directly

Reply:
Clear now :grinning: code updated